### PR TITLE
fix: remove dependencies

### PR DESCRIPTION
**Pull request by MafewTM**
## Description

Removes dependencies as core needs to be nondependent on other resources.

## Checklist

- [ ] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [ ] My code fits the style guidelines.
- [ ] My PR fits the contribution guidelines.

### CODE REVIEW

File: bridge/qb/client/events.lua [hunk 1]
```diff
@@ -7,11 +7,7 @@ end)
 RegisterNetEvent('QBCore:Client:VehicleInfo', function(info)
     local vehicle = NetworkGetEntityFromNetworkId(info.netId)
     local plate = GetPlate(vehicle)
-    local hasKeys = true
-
-    if GetResourceState('qb-vehiclekeys') == 'started' then
-        hasKeys = exports['qb-vehiclekeys']:HasKeys()
-    end
+    local hasKeys = config.hasKeys
```
Comment (Manason) [inline bridge/qb/client/events.lua:10]:
```suggestion
    local hasKeys = config.hasKeys()
```

File: config/server.lua [hunk 2]
```diff
@@ -116,5 +116,17 @@ return {
 
     giveVehicleKeys = function(src, plate)
         exports.qbx_vehiclekeys:GiveKeys(src, plate)
+    end,
+
+    hasKeys = function()
+        exports.qbx_vehiclekeys:HasKeys()
+    end,
```
Comment (Manason) [inline config/server.lua:123]: This is used by the client, but is marked as server config

File: config/server.lua [hunk 3]
```diff
@@ -116,5 +116,17 @@ return {
 
     giveVehicleKeys = function(src, plate)
         exports.qbx_vehiclekeys:GiveKeys(src, plate)
+    end,
+
+    hasKeys = function()
+        exports.qbx_vehiclekeys:HasKeys()
```
Comment (Manason) [inline config/server.lua:122]:
```suggestion
        return exports.qbx_vehiclekeys:HasKeys()
```

File: config/server.lua [hunk 4]
```diff
@@ -116,5 +116,17 @@ return {
...
+    end,
+
+    getAccount = function(jobName)
```
Comment (Manason) [inline config/server.lua:125]:
```suggestion
    getSocietyAccount = function(accountName)
```

File: config/server.lua [hunk 5]
```diff
@@ -116,5 +116,17 @@ return {
...
+        exports.qbx_management:GetAccount(jobName)
```
Comment (Manason) [inline config/server.lua:126]:
```suggestion
        return exports.qbx_management:GetAccount(accountName)
```

File: config/server.lua [hunk 6]
```diff
@@ -116,5 +116,17 @@ return {
...
+    end,
+    
+    removeMoney = function(jobName, payment)
```
Comment (Manason) [inline config/server.lua:129]:
```suggestion
    removeSocietyMoney = function(accountName, payment)
```

File: config/server.lua (hunk 5 above)
Comment (Manason) [reply config/server.lua:126]: default to a direct renewed banking call?

**Review by Manason — CHANGES_REQUESTED** [review]

File: config/server.lua (hunk 2 above)
Comment (MafewTM) [reply config/server.lua]: Fixed.

**Pull request merged**